frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -300,6 +317,46 @@ function FileManagerMenu:setUpdateItemTable()
             },
         }
     }
+    if crashlog.exists then
```

Comment:
Reading this feels like a typo :) We mostly always wrap that and call a function.
So may be store the boolean as `_exists` and have a `function zzz:exists() return _exists end`